spec/models/proxy_performance_spec.rb

```diff
@@ -4,7 +4,37 @@
   let(:site) {create(:site)}
   let(:proxy) {create(:proxy)}
   let(:proxy_performance) do
-    ProxyPerformance.create(:proxy => proxy, :site => site)
+    @proxy_performance ||= ProxyPerformance.create(
```

Comment:
Is there a reason this couldn't be a factory? Also, why the `||=` assignment? Don't you want to recreate the performance object each time `proxy_performance` is evaluated?